pegasus/sites.v3/code.org/public/teacher-dashboard/landing.haml

```diff
@@ -4,8 +4,14 @@ theme: none
 content-type: text/ng-template
 ---
 :ruby
```

Comment:
Now seems like a good time to do so, given that additional methods are being added.